service/routes.py

```diff
@@ -33,10 +27,9 @@ def index():
     Return some useful information about the service, including
     service name, version, and the resource URL
     """
-    url = request.base_url + 'inventories'
```

Comment:
`black` ensures we're using double quotes, so I think this was just auto formatting from `black service/`